website/content/docs/commands/auth/move.mdx

```diff
@@ -0,0 +1,34 @@
+---
+layout: docs
+page_title: auth move - Command
+description: |-
+  The "auth move" command moves an existing auth method to a new path. Any
+  tokens from the old auth method are revoked, but all configuration
+  associated with the engine is preserved.
```

Comment:
```suggestion
  associated with the engine are preserved.
```

Reply:
keeping as `is` since I'm using `configuration` before

Reply:
added this and made it `configurations` actually [ec185fc](https://github.com/hashicorp/vault/pull/14314/commits/ec185fca4801fb73794fecd7dda77eea8a8e6d22)

Reply:
OK. 'all' is inclusive.
So either...
...all configurations are associated with the engine are preserved
or
...the configuration associated with the engine is preserved.

Reply:
roger, i went with the first option